compiler/rustc_feature/src/unstable.rs

```diff
@@ -598,6 +598,8 @@ declare_features! (
     (unstable, strict_provenance, "1.61.0", Some(95228)),
     /// Allows string patterns to dereference values to match them.
     (unstable, string_deref_patterns, "1.67.0", Some(87121)),
+    /// Allows structs to carry target_feature information.
+    (unstable, struct_target_features, "CURRENT_RUSTC_VERSION", Some(129107)),
```

Comment:
While we're at it, could we mark this as incomplete? This is neither RFC accepted, and there seem to be some open FIXMEs that may affect soundness w.r.t. its interactions w/ `target_feature_11`

Reply:
good point - while I don't think it is unsound as it is now but just overly restrictive, the feature is certainly not complete anyway :-) (in fact, if you have good suggestions of how to make it work with generics, I am all ears)